File: SDKs/beezup.com/2.0/typescript/src/sdk/models/shared/exceptiondetail.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";



export class ExceptionDetail extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=helpLink" })
  helpLink?: string;

  @SpeakeasyMetadata({ data: "json, name=message" })
  message?: string;

  @SpeakeasyMetadata({ data: "json, name=stackTrace" })
  stackTrace?: string;

  @SpeakeasyMetadata({ data: "json, name=type" })
  type?: string;
}

File: SDKs/beezup.com/2.0/python/src/sdk/models/shared/exceptiondetail.py
import dataclasses
from typing import Optional
from dataclasses_json import dataclass_json
from sdk import utils


@dataclass_json
@dataclasses.dataclass
class ExceptionDetail:
    help_link: Optional[str] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('helpLink') }})
    message: Optional[str] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('message') }})
    stack_trace: Optional[str] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('stackTrace') }})
    type: Optional[str] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('type') }})
    
